apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md

```diff
@@ -0,0 +1,280 @@
+        AND "bene_id" <='-10000000009999'
+    );
+        
+        
+    DELETE FROM public.carrier_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.dme_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.hha_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    
+    DELETE FROM public.hospice_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.inpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.outpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.snf_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    
+    DELETE FROM public.partd_events
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiary_monthly
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries_history
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';`
+
+
+## Load Synthetic Data in Hosted Environments
+
+1. Before loading data, we need to make sure in TEST, SBX or PROD there are no collisions for the main properties in synthetic data i.e. bene_id, mbi_num, claim_id, etc. Each of the queries should return zero records. Aggregate RIF file data from all claim types, beneficiaries, and Part D Events located in `apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-synthea/` to generate the following queries for the various synthea end-state properties: 
+
+        -- For Beneficiary
+        Select count(*) from public.beneficiaries where mbi_num in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where bene_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where hicn_unhashed in (‘XXX’,’YYY’,…);
+
+        
+        -- For PDE
+        Select count(*) from public.partd_events where pde_id in (‘XXX’,’YYY’,…);
+
+
+        -- For all claim types
+
+        Select count(*) from public.carrier_claims where claim_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.carrier_claims where clm_grp_id BETWEEN ‘XXX’ AND ’YYY’;
+
+
+        -- For DME and Carrier
+        Select count(*) from public.carrier_claims where carr_clm_cntl_num in (‘XXX’,’YYY’,…);
+
+
+        -- For HHA, Hospice, Inpatient, Outpatient, SNF
+        Select count(*) from public.hha_claims where fi_doc_clm_cntl_num in (‘XXX’,’YYY’,…);
```

Comment:
I think we should list out the queries for each type, or make it a union query so that it is one stop shopping.